README.md

```diff
@@ -68,32 +69,32 @@ psql -h localhost -U fence_user -d fence_db
 The official Docker installation page can be found [here](https://docs.docker.com/install/#supported-platforms). If you've never used Docker before, it may be helpful to read some of the Docker documentation to familiarize yourself with containers.
 
 ### Docker Compose Setup
-If you are using Linux, then the official Docker installation does not come with Docker Compose. The official Docker Compose installation page can be found [here](https://docs.docker.com/compose/install/#prerequisites). You can also read an overview of what Docker Compose is [here](https://docs.docker.com/compose/overview/) if you want some extra background information. Go through the steps of installing Docker Compose for your platform, then proceed to setting up credentials.
+If you are using Linux, then the official Docker installation does not come with Docker Compose. The official Docker Compose installation page can be found [here](https://docs.docker.com/compose/install/#prerequisites). You can also read an overview of what Docker Compose is [here](https://docs.docker.com/compose/overview/) if you want some extra background information. Go through the steps of installing Docker Compose for your platform, then proceed to set up credentials.
 
 ### Setting up Credentials
-Setup credentials for fence, a custom root CA  and SSL certs with the provided script by running either:
+Setup credentials for the fence, a custom root CA  and SSL certs with the provided script by running either:
```

Comment:
I'd suggest capitalizing Fence instead of using "the fence"
you can even include a link to the Fence repo if you think that makes it more clear
```suggestion
Setup credentials for Fence, a custom root CA  and SSL certs with the provided script by running either:
```

Reply:
Thanks!